scripts/update_thrift_bindings.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+HIVE_VERSION='2.3'
+
+# Create a temporary directory
+scriptdir=`dirname $0`
+tmpdir=`mktemp -d 2>/dev/null || mktemp -d -t 'pyhive'`
+
+# Copy patch that adds legacy GetLog methods
+cp $scriptdir/thrift-patches/TCLIService.patch $tmpdir
+
+# Download TCLIService.thrift from Hive
+curl https://raw.githubusercontent.com/apache/hive/branch-$HIVE_VERSION/service-rpc/if/TCLIService.thrift > $tmpdir/TCLIService.thrift
```

Comment:
I used https://github.com/apache/hive/blob/rel/storage-release-2.2.1/service-rpc/if/TCLIService.thrift (the latest stable version).